fsd_utils/authentication/decorators.py

```diff
@@ -0,0 +1,65 @@
+from functools import wraps
+from urllib.parse import quote_plus
+
+from flask import abort
+from flask import current_app
+from flask import redirect
+from flask import request
+from fsd_utils.authentication.utils import validate_token_rs256
+from jwt import ExpiredSignatureError
+from jwt import PyJWTError
+
+from .config import config_var_auth_host
+from .config import config_var_user_token_cookie_name
+
+
+def _failed_redirect(message: str = "Link expired or invalid"):
+    authenticator_host = current_app.config.get(config_var_auth_host)
```

Comment:
I wonder if it's best just to do a hard lookup for the key and let the `KeyError` bubble up, or throw a custom exception? This would still come through to the app as a 500, but would be catchable by exception monitoring hooks e.g. Sentry.